### ISSUE TITLE
APKEditor is unable to merged apks (Java heap space)

### ISSUE THREAD

**Issue by raoshaab** [REAndroid/APKEditor#7]
Android Version 12
Apk editor version 1.1.3

While merging the apks

```
$ java -jar APKEditor-1.1.3.jar m -i new/ -o out.apk
00.000 I: [MERGE] Merging ...
   Input: new
 Output: out.apk
 ----------------------------
00.043 I: [MERGE] Searching apk files ...
00.047 I: [MERGE] Found apk files: 11
00.080 [MERGE] ading: split_config.xxhdpi.apk
00.083 I: [MERGE] Found modules: 11
00.084 I: [MERGE] Merging string pools ...
Exception in thread "main" java.lang.OutOfMemoryError: Java heap space
        at com.reandroid.lib.arsc.item.BlockItem.setBytesLength(BlockItem.java:59)
        at com.reandroid.lib.arsc.item.StringItem.onReadBytes(StringItem.java:132)
        at com.reandroid.lib.arsc.base.Block.readBytes(Block.java:38)
        at com.reandroid.lib.arsc.array.OffsetBlockArray.onReadBytes(OffsetBlockArray.java:189)
        at com.reandroid.lib.arsc.base.Block.readBytes(Block.java:38)
        at com.reandroid.lib.arsc.base.BlockContainer.onReadBytes(BlockContainer.java:149)
        at com.reandroid.lib.arsc.chunk.BaseChunk.onReadBytes(BaseChunk.java:60)
        at com.reandroid.lib.arsc.base.Block.readBytes(Block.java:38)
        at com.reandroid.lib.arsc.pool.TableStringPool.readFromTable(TableStringPool.java:74)
        at com.reandroid.lib.apk.ApkModule.getVolatileTableStringPool(ApkModule.java:273)
        at com.reandroid.lib.apk.ApkBundle.mergeStringPools(ApkBundle.java:80)
        at com.reandroid.lib.apk.ApkBundle.mergeModules(ApkBundle.java:43)
        at com.reandroid.apkeditor.merge.Merger.run(Merger.java:55)
        at com.reandroid.apkeditor.merge.Merger.execute(Merger.java:195)
        at com.reandroid.apkeditor.Main.execute(Main.java:64)
        at com.reandroid.apkeditor.Main.main(Main.java:38)

```

**Comment by REAndroid** [comment]
Attach/share your apk files

**Comment by raoshaab** [comment]
Fixed with version 1.1.4 !!
